### ISSUE TITLE
If window is closed when notification is showing, it won't show changed pages when windows is reopened.

### ISSUE THREAD

**Issue by Travellingman** [sneakypete81/updatescanner#460]
Thanks for creating a new issue!

Before continuing, please use the search box in the header above to check if someone has already reported it.

To help explain the issue, please include the following:
 * OS version:
 * Firefox version:
 * Update Scanner version:

#### Steps to reproduce the problem:
[Include webpage URL, if applicable]

#### What actually happened:
[Include a screenshot]

**Comment by Travellingman** [comment]
OS: Windows 10 with all patches, 64bit
FF: 67.0.3
Scanner: 4.4.0

When browing the web, notifications arrived just as I was closing FF.  Upon reopening FF, no notifications appeared, and the "Show All Updates" showed nothing had changed.  With over 100 pages being monitored, finding WHICH had changed is ... boring and a waste of time.

SUGGESTION:  Do not clear the flag that a page has changed until it has been viewed, or provide another means to manually clear such a flag.

**Comment by sneakypete81** [comment]
I've not seen this. Did you click the notification, or did it just go away by itself?

The sites shouldn't be marked as "viewed" when the notification appears, only after you click it to load the changes into the browser tabs.

**Comment by Travellingman** [comment]
I noticed the notification just as I was clicking to close the window.  When I reopened the window to see what had changed, there was no notification.  

I often find updates have occurred WITHOUT seeing any notification.  Sometimes I have gone into the Mozilla profiles and dug them out based on the last modified date and time.

**Comment by sneakypete81** [comment]
Does the number of changes display correctly in the toolbar button?

**Comment by Travellingman** [comment]
Sorry, I can't retest this.  I had to rebuild my computer from scratch recently, and now everything is working as expected.  Maybe a new profile fixed it, maybe something else.

Thanks for looking into it.

Darrell

Please note: message attached

**Comment by sneakypete81** [comment]
Ok, thanks. Closing for now, please reopen if you see this again.
